src/Raven.Studio/typescript/components/common/ButtonGroupWithLabel.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { ButtonGroup, ButtonGroupProps, Label } from "reactstrap";
+
+interface ButtonGroupWithLabelProps extends ButtonGroupProps {
+    label?: string | JSX.Element;
+}
+
+export function ButtonGroupWithLabel(props: ButtonGroupWithLabelProps) {
+    const { children, label, ...rest } = props;
+
+    return (
+        <ButtonGroup className="flex-vertical btn-group-label" data-label={label} {...rest}>
```

Comment:
Could you also check if it works if you pass icon and span as label? If no, then change in ButtonGroupWithLabelProps from string | JSX.Element to string.

Reply:
I believe it's not possible - I've changed it to string.